### PR TITLE
Feature/settings for errors

### PR DESCRIPTION
**Pull request by xavierlefevre**

### CODE REVIEW

File: src/TextInput.js [hunk 1]
```diff
@@ -162,6 +156,7 @@ class FormidableTextInput extends Component {
           />
         </View>
         {this.props.showError &&
+          this.state.errorMessage &&
```
Comment (Almouro) [inline src/TextInput.js]: but then, when there's an error, aren't the next inputs pushed down by the error container?

**Comment by Almouro** [comment]
Hmm, I'd rather set a prop like `validate` and/or `getErrorMessage` on the TextInput for custom behaviour.
What do you think?

The validation is still kind of a mess in this repo, I'll make a first pass at cleaning it tomorrow night, in case you're available :)

**Comment by xavierlefevre** [comment]
@Almouro yes, let's take 30min this evening ! fine by me

**Comment by xavierlefevre** [comment]
@Almouro : what do we do with this ? :)

**Comment by Almouro** [comment]
@xavierlefevre should we close this then?

**Comment by Almouro** [comment]
@xavierlefevre Closing this then, feel free to reopen if need be :)

**Pull request closed without merging**